backend/tests/integration/test_iteration_strategies_integration.py

```
@@ -0,0 +1,485 @@
+    enabled: true
+    connection_params: {}
+        """
+        
+        with tempfile.NamedTemporaryFile(mode='w', suffix='.yaml', delete=False) as f:
+            f.write(config_yaml)
+            temp_path = f.name
+        
+        try:
+            # Should fail during validation
+            loader = ConfigurationLoader(temp_path)
+            with pytest.raises(Exception):  # ValidationError or ConfigurationError
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)